website/content/ja/contribute/projects/localizations/index.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: 翻訳
+title: ローカライゼーション
 list_pages: false
 ---
 
 ## イントロダクション
 
-CNCFの素晴らしさの一端は関わっている人々の多様性とコミュニティからサポートされているからです。このコミュニティをより包括的にする一例として、可能な箇所でコンテンツの翻訳を提供することです。翻訳には当然それを作成およびメンテナンスする母語話者を必要とし、このページはTAG App Deliveryのコンテンツでのやり方を示します。
+CNCFの素晴らしさの一端は関わっている人々の多様性とコミュニティからサポートされているからです。このコミュニティをより包括的にする一例として、可能な箇所でコンテンツのローカライゼーションを提供することです。ローカライゼーションには当然それを作成およびメンテナンスする母語話者を必要とし、このページはTAG App Deliveryのコンテンツでのやり方を示します。
 
 ## ガイドライン
 
-各翻訳で異なりますが、一般的なガイドラインやノウハウは[過去の翻訳PR](https://github.com/cncf/tag-app-delivery/issues?q=label%3Atranslation+)や[Kubernetes SIG-Docsローカライゼーションドキュメント](https://kubernetes.io/ja/docs/contribute/localization/)を参考にすることをおすすめします。
+各ローカライゼーションで異なりますが、一般的なガイドラインやノウハウは[過去のローカライゼーションPR](https://github.com/cncf/tag-app-delivery/issues?q=label%3Atranslation+)や[Kubernetes SIG-Docsローカライゼーションドキュメント](https://kubernetes.io/ja/docs/contribute/localization/)を参考にすることをおすすめします。
```

Comment:
Thank you for your work. Let me suggest a minor fix for the sake of naturalness.

```suggestion
ローカライゼーションごとに異なりますが、一般的なガイドラインやノウハウは[過去のローカライゼーションのPR](https://github.com/cncf/tag-app-delivery/issues?q=label%3Atranslation+)や[Kubernetes SIG-Docsローカライゼーションドキュメント](https://kubernetes.io/ja/docs/contribute/localization/)を参考にすることをおすすめします。
```